script/zonemaster-cli

```diff
@@ -111,17 +108,66 @@ Print the effective profile used in JSON format, then exit.
 
 =over 4
 
-=item B<--test>=TESTCASE, B<--test>=TESTMODULE
+=item B<--test>=EXPR
+
+Select test cases to run (sets the C<test_cases> profile property).
+EXPR is composed of terms (individual test cases, test modules, or C<all>) and
+operators (C<+> to add, C<-> to remove).
+For details, see C<man zonemaster-cli>.
+
+=begin :man
+
+An EXPR starting with a term (e.g., C<all>, C<nameserver01>) completely replaces
+the selection (i.e., the value of the C<test_cases> profile property).
+An EXPR starting with an operator applies a delta to the selection.
+An empty EXPR (C<--test=>) leaves the selection unchanged (formally, it is
+updated by applying the identity transformation).
```

Comment:
What does "leaves the selection unchanged" mean? For what can I use that?

Reply:
`--test=` is useful the same way `x + 0` is useful. I don't expect you to type it out on the command line, but it could make life easier if you want to call zonemaster-cli programmatically.

Reply:
Can you please clarify that, preferably below when it listed on its own line.